fe/fe-core/src/main/java/com/starrocks/sql/parser/AstBuilder.java

```diff
@@ -895,6 +1069,15 @@ public ParseNode visitFunctionCall(StarRocksParser.FunctionCallContext context)
             return new TimestampArithmeticExpr(functionName, e3, e2, e1.getValue());
         }
 
+        if (functionName.equalsIgnoreCase("isnull")) {
```

Comment:
Is there any plan to refactor these if-else into a map?

Reply:
There is no such plan, because ifnull does not have a keyword, but in the grammar of starrocks, it has to be parsed into IsNullPredicate, which requires special judgment. It cannot be put into a map to reuse common logic like other grammars